system/os_platform/win32_sdl/CMakeLists.txt

```diff
@@ -0,0 +1,132 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
do you use this file? I wanted to remove this one (mac_sdl/CMakeLists.txt and mac_sdl/build.sh), it was just for testing. All projects/applications should have their own cmakelists file in their directory. Currently it is a little bit messy, but in future I would like to have single cmake file to build the application for all platforms (right now there are shell scripts for arm builds)

Reply:
Well I made my tests with this file and then discovered you created dedicated CMakeLists files for each application ;-)
I will have to adapt the CMakeLists files to switch between Win32 and Mac hal according to the platform.
I'll create a separate PR for that if you're OK

Reply:
Well I ended up making the change in gabosc specific CMakeList file : added a if(WIN32) for win32 hal files + math.h override.

Reply:
I also added elfstrip binary for win32 + a CMakeLists file to generate it's project